juriscraper/AbstractSite.py

```diff
@@ -18,6 +19,13 @@
 from juriscraper.lib.test_utils import MockRequest
 
 logger = make_default_logger()
+phantomjs_executable_path = '/usr/local/bin/phantomjs'
+_legacy_path = "/usr/local/phantomjs/phantomjs"
+if os.path.isfile(_legacy_path) and os.access(_legacy_path, os.X_OK):
+    phantomjs_executable_path = _legacy_path
```

Comment:
This looks good to me. Want to remove this comment and then add a deprecation warning here too? That'll:

 - fix the issue
 - default to the better path
 - tell people they should change something

If we want to be super thorough we could link to a commit/issue/pr/etc on Github in the deprecation warning so people see the discussion, but that feels like overkill to me.

Reply:
Done!  The overkill was just a simple sentence, and a browser tab was open anyway.